CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 0.10.0 (12/29/2016)
+
+### Breaking bug fixes
+
+* Fix oauth2 token refresh (#109) ([@ianmetcalf][])
+
+### Backwards compatible changes
+
+* Remove unused dependencies and upgrade remaining dependencies. (#112) ([@jmdobry][])
```

Comment:
I went ahead and reverted to `"request": "^2.74.0"` so that decisions about `request v2.79.0` and Node.js 0.10/0.12 don't have to block this release.